zap/src/main/java/org/zaproxy/zap/extension/script/JavaScriptEngineWrapper.java

```diff
@@ -25,7 +25,7 @@
 import org.fife.ui.rsyntaxtextarea.SyntaxConstants;
 import org.parosproxy.paros.view.View;
 
-public class JavascriptEngineWrapper extends DefaultEngineWrapper {
+public class JavaScriptEngineWrapper extends DefaultEngineWrapper {
```

Comment:
API?

I'm happy w/ dropping this; or splitting out the human readable part of this commit from the class stuff.

`JavaScript` is the brand, `javascript` is acceptable, `Javascript` is just wrong.

Reply:
`Javascript` matches proper capitalization for Java class names.